bridges/SteamCommunityBridge.php

```diff
@@ -56,12 +57,17 @@ public function getName() {
 	}
 
 	public function getURI() {
+		if ($this->getInput('category') == 'workshop')
+			return self::URI . '/workshop/browse/?appid='
+				. $this->getInput('i') . '&browsesort=mostrecent';
+
 		return self::URI . '/app/'
 			. $this->getInput('i') . '/'
-			. $this->getInput('category');
+			. $this->getInput('category')
+			. '/?p=1&browsefilter=mostrecent';
 	}
 
-	public function collectData() {
+	public function collectMedia() {
```

Comment:
This function should be private. In fact, any function that is not part of the API needs to be protected or private.

```suggestion
	private function collectMedia() {
```